test-network-function/suite_test.go

```diff
@@ -82,6 +84,18 @@ func createClaimRoot() *claim.Root {
 	}
 }
 
+func loadJunitXMLIntoMap(result map[string]interface{}, filepath, key string) {
+	var err error
+	if key == "" {
+		var extension = path.Ext(filepath)
```

Comment:
Accidentally replaced filepath.Ext with path.ext. No real impact when running on linux. Will fix it anyways in the next commit.